test/core.jl

```diff
@@ -12,3 +12,26 @@
 
 @test ProgressMeter.Progress(5, "Progress:", Int16(5)).offset == 5
 @test ProgressMeter.ProgressThresh(0.2, "Progress:", Int16(5)).offset == 5
+
+# Performance test (from #171)
+function prog_perf(n)
+    prog = Progress(n)
+    x = 0.0
+    for i in 1:n
+        x += rand()
+        next!(prog)
+    end
+    return x
+end
+
+function noprog_perf(n)
+    x = 0.0
+    for i in 1:n
+        x += rand()
+    end
+    return x
+end
+
+prog_perf(1)
+noprog_perf(1)
+@test @elapsed(prog_perf(10^7)) < 40*@elapsed(noprog_perf(10^7))
```

Comment:
I had to change this constant from `40` to `50` to get the tests to pass on my laptop with the current `release-1.6` branch of julia.  On julia `1.5.3` the tests pass without this change.  Should this constant be adjusted, or does this indicate something wrong to you?

Reply:
Yes, something seems wrong. For me, Julia 1.5 runs in about 70% of the time of Julia master.

Reply:
It looks mostly like a locking issue. If you profile, set the interval to 0.01s (the default of 0.001s interferes too much).